neo/Trie/MPT/MPTNode/BranchNode.cs

```diff
@@ -1,38 +1,41 @@
+using Neo.IO;
 using Neo.IO.Json;
 using System.IO;
+using System.Linq;
 
 namespace Neo.Trie.MPT
 {
     public class BranchNode : MPTNode
     {
+        protected override NodeType Type => NodeType.BranchNode;
         public const int ChildCount = 17;
         public MPTNode[] Children = new MPTNode[ChildCount];
+        public override int Size => base.Size + Children.Sum(n => n.IsEmptyNode ? 1 : 33);
```

Comment:
sum n.Size?

Reply:
Not `child.Size`. If so, we will get the size of the whole tree. We want the size of one node.
We fold the child tree to an `HashNode`.

Reply:
Got it. Could you add some UT (if there are not) checking that the size it's the same of the serialization result?

Reply:
Will do.

Reply:
done